uamqp/message.py

```diff
@@ -235,6 +235,33 @@ def delivery_annotations(self):
     def delivery_annotations(self, value):
         self._delivery_annotations = value
 
+    @property
+    def data(self):
+        if not self._message or not self._body:
+            return None
+        # pylint: disable=protected-access
+        if self._body.type == c_uamqp.MessageBodyType.DataType:
+            return self._body.data
+        return None
+
+    @property
+    def sequence(self):
```

Comment:
@annatisch - currently this property is returning the same generator each time. After it's been used once, it returns None. Do you know if there's any way to have it return a new generator each time, like `get_data` does?

Reply:
update: this is not being used up, and works when called multiple times.